hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/transfer/ClassicTransfersDecoder.java

```diff
@@ -224,6 +229,26 @@ public TransactionBody decodeHrcTransferNftFrom(
     }
 
     public ResponseCodeEnum checkForFailureStatus(@NonNull final HtsCallAttempt attempt) {
+        // match mono behaviour
+        if (Arrays.equals(attempt.selector(), ClassicTransfersTranslator.TRANSFER_TOKENS.selector())) {
+            final var call = ClassicTransfersTranslator.TRANSFER_TOKENS.decodeCall(attempt.inputBytes());
+            final Address[] accountIds = call.get(1);
+            if (accountIds.length == 0) {
+                return ResponseCodeEnum.EMPTY_TOKEN_TRANSFER_ACCOUNT_AMOUNTS;
+            }
+            final var tokenStore = attempt.enhancement().nativeOperations().readableTokenStore();
+            final var token = tokenStore.get(ConversionUtils.asTokenId(call.get(0)));
+            if (token == null) {
+                return ResponseCodeEnum.SPENDER_DOES_NOT_HAVE_ALLOWANCE;
```

Comment:
The "SPENDER_DOES_NOT_HAVE_ALLOWANCE" obviously makes no sense, but this is the current mono behaviour for `transferTokens`.
Do we want to still do the matching for such cases at this point, where we are close to modularized codebase release @tinker-michaelj